src/player.py

```diff
@@ -1,2 +1,7 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name):
```

Comment:
You might want to add current_room to this class (and every instantiation of it).